### PR TITLE
[FEATURE] Improve Cancel button clarity

### PR DESCRIPTION
**Pull request by Kuzirashi**
# V1
3e02696961ddd823de0f825012ce4b54

# V2
screenshot-localhost-8080-2018 09 18-11-28-49

### CODE REVIEW

**Comment by Bagaric** [comment]
Please capitalize the letters in the enumerated bullets - otherwise looks good.

**Comment by Bagaric** [comment]
Maybe also **bold** important parts for clarity.

File: app/components/TransactionScanner/TransactionDetails.js [hunk 1]
```diff
@@ -207,37 +210,67 @@ class TransactionDetails extends Component {
...
     const isOwner = this.isOwner(transaction);
 
-    if (isOwner && !isFrozen && status === TRANSACTION_STATUS.SCHEDULED) {
-      return (
-        <div className="d-inline-block text-center mt-2 mt-sm-5 col-12 col-sm-6">
+    const cancelButtonEnabled = isOwner && !isFrozen && status === TRANSACTION_STATUS.SCHEDULED;
+
+    return (
+      <div className="alert alert-info">
+        In order to cancel this transaction please switch to account {transaction.owner}{' '}
```
Comment (Bagaric) [inline app/components/TransactionScanner/TransactionDetails.js]: Please show this part only when the user is indeed not the owner.

**Comment by kosecki123** [comment]
I suggest to keep this information pane always visible and add 4th condition, you are an owner of this transaction.

**Pull request merged**